src/features/embalm/stepContent/components/SuccessfulCreateSarcophagus.tsx

```diff
@@ -12,87 +15,168 @@ export function SuccessfulCreateSarcophagus({
 }: SuccessData) {
   const navigate = useNavigate();
   const dispatch = useDispatch();
+  const networkConfig = useNetworkConfig();
 
   const goToDashboard = () => {
     dispatch(goToStep(Step.NameSarcophagus));
     navigate('/dashboard');
   };
 
+  const boxBorder = '0.5px solid';
+
+  const { onCopy: copyShardsId, hasCopied: hasCopiedShardsId } = useClipboard(
+    successEncryptedShardsTxId,
+    { timeout: 5000 }
+  );
+  const { onCopy: copyFileId, hasCopied: hasCopiedFileId } = useClipboard(
+    successSarcophagusPayloadTxId,
+    { timeout: 5000 }
+  );
+
+  const copyIcon = (hasCopiedFlag: boolean, onClick: () => void) =>
+    hasCopiedFlag ? (
+      <CheckIcon mt={4} />
+    ) : (
+      <CopyIcon
+        mt={4}
+        cursor="pointer"
+        bgColor={'transparent'}
+        color="white"
+        aria-label="copy"
+        onClick={onClick}
+      />
+    );
+
+  const arweaveTxId = (txId: string) => (
+    <Text
+      width="50%"
+      fontSize="10px"
+      color="brand.700"
+      textAlign="right"
+      mt={4}
+    >
+      {`${txId.slice(0, 5)}...${txId.slice(-4)}`}
+    </Text>
+  );
+
   return (
     <Flex
-      mt={6}
       direction="column"
       justifyContent="center"
       alignItems="center"
     >
+      {/*
+       * SUCCESS MESSAGE
+       */}
       <Heading
         fontSize="xl"
-        mb={4}
-      >
-        Sarcophagus Creation Successful!
-      </Heading>
-      <Heading
-        fontSize="md"
-        my={4}
+        mx={5}
+        mb={6}
       >
-        Transaction Details
+        Sarcophagus Created Successfully
       </Heading>
+      <SuccessIcon
+        width={50}
+        height={50}
+        mb={6}
+      />
+
+      {/*
+       * DETAILS BOX AND CONTENT
+       */}
```

Comment:
I've tried to add comments in the jsx portion of the component like this in places but I haven't been very consistent. I find them very useful.

Reply:
Yeah jsx gets pretty unwieldy. I haven't been consistent in using these "section" comments at all either tbh, but would be nice to have them used more and more, maybe even adding them to older files whenever a team member messes with it during a task

Reply:
Could we get prettier to leave that one a single line? (if prettier is what's going that). I wouldn't be opposed to some kind of JSX commenting pattern